addon/validators/presence.js

```diff
@@ -16,7 +19,15 @@ export default function validatePresence(options) {
   }
 
   return (key, value, _oldValue, changes, content) => {
-    if (targets && !targets.some((target) => changes[target] || (changes[target] === undefined && content[target]))) {
+    if (
+      targets &&
+      !targets.some((target) => {
+        const change = nestedLookup(changes, target);
```

Comment:
Thank you!  Btw, could we use `Ember.get` for this instead of `nestedLookup`?

Reply:
Done :ok_hand: